functions/index.js

```diff
@@ -65,7 +65,15 @@ exports.createNewTask = functions.https.onRequest((request, response) => {
 });
 
 function getTaskId() {
-
+    dashboardDataset.forEach(element => {
+        if (category == Development) {
+            return element.totalDevelopmentTask + 1;
+        } else if (category == Business) {
```

Comment:
here category is not defined yet
Im not getting how to define

Reply:
yeah im trying to do this.
And I can store these data after completion of new sprint page. Actually first I have print these data into UI after that the data will be fetched into db